actors/common/src/main/java/org/sunbird/learner/actors/skill/SkillmanagementActor.java

```diff
@@ -62,10 +61,65 @@ public void onReceive(Request request) throws Throwable {
         .getOperation()
         .equalsIgnoreCase(ActorOperations.GET_SKILLS_LIST.getValue())) {
       getSkillsList();
+    } else if (request.getOperation().equalsIgnoreCase(ActorOperations.UPDATE_SKILL.getValue())) {
+      updateSkill(request);
     } else {
       onReceiveUnsupportedOperation(request.getOperation());
     }
   }
+  /** Method will Update the skills from user ... */
+  private void updateSkill(Request actorMessage) {
+    ProjectLogger.log(
+        "SkillmanagementActor-updateSkill called",
+        actorMessage.getRequest(),
+        LoggerEnum.INFO.name());
+
+    String userId = (String) actorMessage.getRequest().get(JsonKey.REQUESTED_BY);
+    List<String> list = (List<String>) actorMessage.getRequest().get(JsonKey.SKILL_NAME);
+    CopyOnWriteArraySet<String> skillset = new CopyOnWriteArraySet<>(list);
+    List<Map<String, Object>> correlatedObject = new ArrayList<>();
+    if (StringUtils.isBlank(userId)) {
+      throw new ProjectCommonException(
+          ResponseCode.endorsedUserIdRequired.getErrorCode(),
+          ResponseCode.endorsedUserIdRequired.getErrorMessage(),
+          ResponseCode.CLIENT_ERROR.getResponseCode());
+    }
+    Map<String, Object> filter = new HashMap<>();
+    filter.put(JsonKey.USER_ID, userId);
+    List<String> fields = new ArrayList<>();
+    fields.add(JsonKey.SKILLS);
+    Map<String, Object> result =
+        ElasticSearchUtil.complexSearch(
+            createESRequest(filter, null, fields),
```

Comment:
Can you use ElasticSearchUtil.createSearchDTO instead of using createESRequest? You can implement ElasticSearchUtil.createSearchDTO based on Util.createSearchDto.